src/gui/editors/SongEditor.cpp

```diff
@@ -837,17 +832,49 @@ void SongEditor::updatePositionLine()
 
 
 
+//! Convert zoom slider's value to bar width in pixels
+int SongEditor::calculatePixelsPerBar() const
+{
+	// What we need to raise 2 by to get MIN_PIXELS_PER_BAR and MAX_PIXELS_PER_BAR
+	double minExp = log2(MIN_PIXELS_PER_BAR);
+	double maxExp = log2(MAX_PIXELS_PER_BAR);
+	double exponent = m_zoomingModel->value() / double(ZOOM_STEPS) * (maxExp - minExp) + minExp;
+
+	double ppb = pow(2, exponent);
```

Comment:
```suggestion
	static const double minExp = std::log2(MIN_PIXELS_PER_BAR);
	static const double maxExp = std::log2(MAX_PIXELS_PER_BAR);
	double exponent = m_zoomingModel->value() / static_cast<double>(ZOOM_STEPS) * (maxExp - minExp) + minExp;

	double ppb = std::pow(2, exponent);
```
Can use `static const` local variables because they only need to be calculated once.
Using `static_cast` instead of a functional-style cast.
Added `std::` prefix to math functions.